src/services/exclusionService.ts

```
@@ -0,0 +1,288 @@
+
+      return patterns;
+    } catch {
+      return [];
+    }
+  }
+
+  /**
+   * Convert glob pattern to RegExp
+   * Supports *, **, ? wildcards
+   *
+   * @param pattern - Glob pattern (e.g., "*.log", "**​/node_modules/**")
```

ESLint

[error] Irregular whitespace not allowed.

(no-irregular-whitespace)